app/admin/billing/account.rb

```diff
@@ -4,7 +4,27 @@
   acts_as_safe_destroy
   acts_as_audit
   acts_as_clone
-
+  acts_as_async_destroy('Account')
+  acts_as_async_update('Account',
+                       lambda do
+                         {
+                         contractor_id: Contractor.all.map { |c| [c.name, c.id] },
```

Comment:
I propose to simplify this `all.map {...}` everywhere to: 
```ruby
MyModel.pluck(:name, :id)
```

Reply:
Done